canvas_modules/common-canvas/src/common-properties/controls/datefield/datefield.jsx

```diff
@@ -83,7 +83,7 @@ class DatefieldControl extends React.Component {
 					value={this.value}
 					labelText={this.props.controlItem}
 					hideLabel={this.props.tableControl}
-					light={this.props.controller.getLight()}
+					light={this.props.controller.getLight() && !this.props.control.light}
```

Comment:
What is the difference between both values?

Reply:
@nmgokhale the controller.getLight() is set by the host and controls showing a white background with controls having a grey background, and vice versa, grey background for panel with white background controls.

The props.control.light is always false, only goes to true when controls are rendered within tearsheet. Since the tearsheet is always a grey background, we set the light attribute to false in order to force render white backgrounds on tearsheet.